supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc

```diff
@@ -392,22 +392,43 @@ allowed for use across all distribution documentation:
 
 *See also*: xref:okd[OKD]
 
-[discrete]
 [[operator]]
 ==== Operator
 *Description*: An Operator is a method of packaging, deploying, and managing a
```

Comment:
Instead of duplicating the information exactly, how about we just add a link to the general guidelines?

I feel like we might be setting ourselves up for failure if we duplicate it, and then update it one place but forget to do the other.

We do something like this already in a few places (even in the openshift file already).

```
[discrete]
==== node (noun)
See xref:node[node] in the _General Conventions_ part.
```

These duplicated ones for some reason don't have defined IDs (and they should), but we can take care of fixing that later.

Reply:
Yeah, I wasn't sure how we wanted to handle that, and didn't notice that example of cross-reffing that way. So yep, done. I've just cross-reffed in the same way.